src/rules.jl

```diff
@@ -518,6 +530,45 @@ function lrp!(Rᵏ, rule::GeneralizedGammaRule, layer, modified_layers, aᵏ, R
     @. Rᵏ = aᵏ⁺ * (cˡ⁺ + cʳ⁻) + aᵏ⁻ * (cˡ⁻ + cʳ⁺)
 end
 
+"""
+    LayerNormRule(affine_rule=ZeroRule())
+
+LayerNormRule rule.
+
+# Optional arguments
+- `affine_rule`: Rule applied to affine transformation, defaults to `ZeroRule()`.
+
+# References
+- 
+"""
+struct LayerNormRule{T <: AbstractLRPRule} <: AbstractLRPRule
+    affine_rule::T
+end
+
+LayerNormRule() = LayerNormRule(ZeroRule())
+
+function detached_normalise(aᵏ, layer)
+    dims = 1:length(layer.size)
+    eps = convert(float(eltype(aᵏ)), layer.ϵ)
+    μ = mean(aᵏ, dims = dims)
+    σ = ignore_derivatives(std(aᵏ, dims=dims, mean=μ, corrected=false))
+    return @. (aᵏ - μ) / (σ + eps)
+end
+
+function lrp!(Rᵏ, rule::LayerNormRule, layer::LayerNorm, modified_layer, aᵏ, Rᵏ⁺¹)
+    # forward pass: split in normalization and scale
+    # aᵏ ->(normalize) aᵏₙ ->(scale) aᵏ⁺¹
+    ## normalize
+    aᵏₙ, back = Zygote.pullback(x -> detached_normalise(x, layer), aᵏ)
```

Comment:
I'd call this `aᵏ⁺¹ₙ` to indicate that it corresponds to a modified output of the layer.